utils/images.ts

```diff
@@ -1,8 +1,20 @@
 import { Maybe } from './types';
+import { btoaUniversal } from './strings';
 
-const validExtensions = ['jpg', 'jpeg', 'png', 'gif'];
+type validFits = 'cover' | 'contain' | 'fill' | 'inside' | 'outside';
```

Comment:
Nit: capitalized and singular (`Fit`)

Reply:
Ah! Good catch. The `validExtensions` array right above tricked me!